ksqldb-tools/src/test/java/io/confluent/ksql/tools/migrations/util/CommandParserTest.java

```diff
@@ -273,17 +274,39 @@ public void shouldParseConnectorStatements() {
         + "    \"topic.prefix\"='jdbc-',\n"
         + "    \"table.whitelist\"='users',\n"
         + "    \"key\"='username');";
-    final String dropConnector = "DROP CONNECTOR `jdbc-connector`;";
 
     // When:
-    List<SqlCommand> commands = CommandParser.parse(createConnector + dropConnector);
+    List<SqlCommand> commands = CommandParser.parse(createConnector);
 
     // Then:
-    assertThat(commands.size(), is(2));
-    assertThat(commands.get(0), instanceOf(SqlConnectorStatement.class));
+    assertThat(commands.size(), is(1));
+    assertThat(commands.get(0), instanceOf(SqlCreateConnectorStatement.class));
     assertThat(commands.get(0).getCommand(), is(createConnector));
-    assertThat(commands.get(1), instanceOf(SqlConnectorStatement.class));
-    assertThat(commands.get(1).getCommand(), is(dropConnector));
+    assertThat(((SqlCreateConnectorStatement) commands.get(0)).getName(), is("`jdbc-connector`"));
+    assertThat(((SqlCreateConnectorStatement) commands.get(0)).isSource(), is(true));
+    assertThat(((SqlCreateConnectorStatement) commands.get(0)).getProperties().size(), is(6));
+    assertThat(((SqlCreateConnectorStatement) commands.get(0)).getProperties().get("connector.class"), is("io.confluent.connect.jdbc.JdbcSourceConnector"));
+    assertThat(((SqlCreateConnectorStatement) commands.get(0)).getProperties().get("connection.url"), is("jdbc:postgresql://localhost:5432/my.db"));
+    assertThat(((SqlCreateConnectorStatement) commands.get(0)).getProperties().get("mode"), is("bulk"));
+    assertThat(((SqlCreateConnectorStatement) commands.get(0)).getProperties().get("topic.prefix"), is("jdbc-"));
+    assertThat(((SqlCreateConnectorStatement) commands.get(0)).getProperties().get("table.whitelist"), is("users"));
+    assertThat(((SqlCreateConnectorStatement) commands.get(0)).getProperties().get("key"), is("username"));
+  }
+
+  @Test
+  public void shouldParseDropConnectorStatement() {
+    // Given:
+    final String dropConnector = "DROP CONNECTOR `jdbc-connector`;";
```

Comment:
Can we add some whitespace after the connector name to check that the regex properly leaves out whitespace between the connector name and the semicolon? It's not clear to me from reading the regex whether the greediness of the matching groups is such that this currently works or not.
```suggestion
    final String dropConnector = "DROP CONNECTOR `jdbc-connector` ;";
```

Reply:
Good catch, it did not leave out whitespace. Updated the regex so that it checks for either identifiers with no space and backquoted/double quoted identifiers with pretty much anything inside. I think that should cover all the cases.

Reply:
Hm, ideally there'd be a way to tell the whitespace portion of the original regex (after the connector name and before the semicolon) to be greedy (alternatively, tell the connector name to be un-greedy) to avoid needing to hardcode into the regex the fact that connector names can be wrapped in backticks, double quotes, or neither. I think this should be possible but I haven't played around with it. If you give it a try and it doesn't work then this approach is fine too.

Reply:
If we leave the regex as is (with the fact that connector names can be wrapped in backticks, double quotes, or neither encapsulated in the regex), let's add a comment to explain the regex since it's really gnarly/difficult to parse otherwise.

Reply:
:+1: that ended up working!